### PR TITLE
add list of currently active maintainers

### PR DESCRIPTION
**Pull request by D3DeFi**
I am loosing track of who has push privileges :)

### CODE REVIEW

**Review by pyrodie18 — APPROVED** [review]
Looks good

**Comment by D3DeFi** [comment]
we will have one additional maintainer joining, so will merge after that :)

**Comment by D3DeFi** [comment]
welcome everyone! :grinning:

**Pull request merged**
